src/TLuaInterpreter.cpp

```diff
@@ -2361,6 +2361,25 @@ int TLuaInterpreter::disableKey( lua_State *L )
     return 1;
 }
 
+int TLuaInterpreter::killKey( lua_State *L )
+{
+    string luaSendText="";
+    if( ! lua_isstring( L, 1 ) )
+    {
+        lua_pushstring( L, "killKey: wrong argument type" );
```

Comment:
I don't see why can't we use ``lua_pushfstring()`` - @SlySven ? I'd prefer it over having to wrap things in .arg().